src/frontend/src/lib/components/hero/HeroContent.svelte

```diff
@@ -66,11 +65,9 @@
 		</div>
 	{/if}
 
-	{#if actions}
-		<div transition:slide={SLIDE_PARAMS} class="flex w-full justify-center text-left">
-			<Actions />
-		</div>
-	{/if}
+	<div transition:slide={SLIDE_PARAMS} class="flex w-full justify-center text-left">
```

Comment:
ditto about the transition

Reply:
in this case, the if is a few levels above, where a parent has already a `slide`, so we need to add `|local` to avoid concurrency